bzt/engine.py

```diff
@@ -456,11 +456,8 @@ def find_file(self, filename):
             parsed_url = parse.urlparse(filename)
             downloader = ExceptionalDownloader(self.get_http_client())
             self.log.info("Downloading %s", filename)
-            tmp_f_name, http_msg = downloader.get(filename)
-            cd_header = http_msg.get('Content-Disposition', '')
-            dest = cd_header.split('filename=')[-1] if cd_header and 'filename=' in cd_header else ''
-            if not dest:
-                dest = os.path.basename(parsed_url.path)
+            tmp_f_name = downloader.get(filename)
+            dest = os.path.basename(parsed_url.path)
```

Comment:
This is regression. Some webservers serve files from URLS like `http://abc.com/get.php?fileId=1234` while true file name for download is sent via `Content-Disposition` header.